classes/log4j2.properties

```diff
@@ -0,0 +1,14 @@
+appenders=console
```

Comment:
@yuhai For this pull, we should call it "rename" as it actually changed the content of log4j.properties, please reword.

Thanks.